src/saveSvgAsPng.js

```diff
@@ -19,6 +19,7 @@
   const isElement = obj => obj instanceof HTMLElement || obj instanceof SVGElement;
   const requireDomNode = el => {
     if (!isElement(el)) throw new Error(`an HTMLElement or SVGElement is required; got ${el}`);
+    return;
```

Comment:
Why are these `return` statements necessary?

Reply:
They aren't, just experiment cruft. Sorry about that!